tests/data/fdp_invalid_missingRequired.ttl

```diff
@@ -1,11 +1,11 @@
 @prefix ex: <http://example.com/ns#> .
 @prefix fdp: <http://rdf.biosemantics.org/ontologies/fdp-o#> .
-@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix dct: <http://purl.org/dc/terms/> .
```

Comment:
I am not sure what is more common, if using the prefix `dct` or `dcterms`. In practice it doesn't make any difference, but I think I have seen `dcterms` more frequently. What do you think?

Reply:
You're right. We should use `dcterms` to be consistent with the extended [default namespaces](https://rdflib.readthedocs.io/en/stable/_modules/rdflib/namespace.html) in the library `rdflib v5.0`. The rdflib v5.0 was released a month ago. We're currently still using `rdflib v4.2.2` which does not have the namespace 'dcterms'.

Reply:
Ah good find. Maybe we should upgrade to 5.0.